contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -5059,8 +5065,7 @@ exec_stmt_execsql(PLtsql_execstate *estate,
 		/* If current plan constains a pivot operator, we remove the plan */
```

Comment:
This comment does not match the code anymore. Please update comment.

Reply:
Sure